### DISCUSSION TITLE
Differentiate method and class in Apex Test section

### DISCUSSION THREAD

**Discussion by djmortensen** [forcedotcom/salesforcedx-vscode#5861] [Ideas]
**Is your feature request related to a problem? Please describe.**
When I open the Apex Tests in the dropdown, I have a large amount of tests in there as all test classes are expanded to reveal all tests within each class. I can be scrolling for ages because to find the test I need. Similarly, there is no way to differentiate between class and test method and therefore it can be difficult to distinguish where I am in the larger list of classes that is alphabetised.  

**Describe the solution you'd like**
I would like an expand all/collapse all button in the upper right of the apex tests dropdown to be able to view all test classes expanded or open. Also, I think the test class name should be one color and the test method name be another so as to easily distinguish class from method when expanded. 

**Describe alternatives you've considered**

**Additional context**

![Screenshot 2024-02-23 at 10 41 00](https://github.com/forcedotcom/salesforcedx-vscode/assets/131677066/31ce1aee-c387-4fd0-8d3a-6edd9662b352)

**Comment by git2gus** [comment]
This issue has been linked to a new work item: W-15109263

**Comment by daphne-sfdc** [comment]
Hi @daniel-mortensen-ovo, Thank you for this feature request.  We'll let you know when we implement it to improve your user experience.

**Comment by diyer** [comment]
This is partially implemented. We have collapse/expand all for Apex tests but not the color differentiator

**Reply by djmortensen** [reply]
Great! When was this implemented?
